Packs/Uptycs/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Uptycs
+- updated Uptycs jwt token authentication and added new commands.
```

Comment:
Please add a list of the new commands, as in the example in the screenshot.
![image](https://github.com/demisto/content/assets/71597826/119d7c38-396c-4f6d-ad91-51db24f290b3)

Reply:
updated